addons/project/report/project_report_views.xml

```diff
@@ -28,16 +28,23 @@
             <field name="model">report.project.task.user</field>
             <field name="arch" type="xml">
                 <search string="Tasks Analysis">
+                    <field name="project_id"/>
                     <field name="name" />
+                    <field name="stage_id"/>
+                    <field name="user_id"/>
+                    <field name="partner_id" filter_domain="[('partner_id', 'child_of', self)]"/>
+                    <field name="name" string="Tags" filter_domain="[('task_id.tag_ids', 'ilike', self)]"/>
                     <field name="date_assign"/>
                     <field name="date_end"/>
                     <field name="date_deadline"/>
                     <field name="date_last_stage_update"/>
-                    <field name="project_id"/>
-                    <field name="user_id"/>
-                    <field name="partner_id" filter_domain="[('partner_id', 'child_of', self)]"/>
-                    <field name="stage_id"/>
-                    <filter string="Unassigned" name="unassigned" domain="[('user_id', '=', False)]"/>
+                    <filter string="My Projects" name="own_projects" domain="[('project_id.user_id', '=', uid)]"/>
+                    <filter string="My Tasks" name="my_tasks" domain="[('task_id.user_id', '=', uid)]"/>
+                    <separator/>
+                    <filter string="Starred" name="starred" domain="[('task_id.priority', 'in', [1, 2])]"/>
```

Comment:
As stated in report.project.task.user I am unsure this domain still has sense as the project.task model does not have a priority of value 2.

Reply:
@IT-Ideas Done. Thanks :)